### ISSUE TITLE
sky class in apollo for semantic segmentation 

### ISSUE THREAD

**Issue by jarvisWang0903** [ApolloAuto/apollo#9176]
**Describe the bug**
For "sky" class the Class_Id to train_Id in your description is 17 : 0, However, I did not find the annotation of sky's class id when I actually processed the image

**To Reproduce**
1. I processed the img to mapping the cityscapes,but i did't found the sky class in your dataset, but in the description it does has, did you forget to label it or did I deal with it?
2. Dataset: road02_seg/ColorImage
3. Label:  road02_seg/Label

**Screenshots**
In your website the label is like 
![apollo1](https://user-images.githubusercontent.com/35374455/61516748-34aaa580-aa38-11e9-9178-396061d483b4.png)
But it seems like this when i process your dataset, It looks like the sky class is missing
![apollo_lbl_02_2](https://user-images.githubusercontent.com/35374455/61516796-50ae4700-aa38-11e9-8994-e8f679e84673.png)

**Comment by TillBeemelmanns** [comment]
Not only road02_seg does not contain the sky class, but also road03_seg and road04_seg are affected

**Comment by jarvisWang0903** [comment]
@techoe Help~

**Comment by jarvisWang0903** [comment]
> Not only road02_seg does not contain the sky class, but also road03_seg and road04_seg are affected. So basically the whole semantic segmentation dataset does not contain the sky class

Yes, But it does have in there description. Did they forget to label this class? And I also find that the rider class includes motorcycles/bicycle and rider, so it has to be ignored with sky class when I process to match the cityscapes 19 class.
